tests/test_xnat2bids.py

```diff
@@ -1,48 +1,121 @@
+            '--mem 16000', 
+            '--nodes 1', 
+            '--cpus-per-task 2', 
+            '--job-name xnat2bids', 
+            '--output /gpfs/scratch/%u/logs/xnat2bids_test%J.log', 
+            '--mail-user example-user@brown.edu', 
+            '--mail-type ALL']
+
+        bindings_validation_set = [
+            "/gpfs/data/bnc/shared/bids-export/",
+            "/gpfs/data/bnc/shared/scripts/fmcdona4/bidsmap.json"
+        ]
+
+        # Run compile_xnat2bids_list()
+        x2b_param_list, bindings = compile_xnat2bids_list(session, test_params, user)
     
-    # Initialize validation sets
-    x2b_validation_set = [
-    'XNAT_E00152', 
-    '/gpfs/data/bnc/shared/bids-export/', 
-    '--host https://xnat.bnc.brown.edu', 
-    '--session-suffix -1', 
-    '--bidsmap-file /gpfs/data/bnc/shared/scripts/fmcdona4/bidsmap.json',
-    '--includeseq 1 --includeseq 2', 
-    '--skipseq 3', 
-    '--overwrite']
-
-
-    slurm_validation_set = [
-        '--time 04:00:00', 
-        '--mem 16000', 
-        '--nodes 1', 
-        '--cpus-per-task 2', 
-        '--job-name xnat2bids', 
-        '--output /gpfs/scratch/%u/logs/xnat2bids_test%J.log', 
-        '--mail-user example-user@brown.edu', 
-        '--mail-type ALL']
-
-    bindings_validation_set = [
-        "/gpfs/data/bnc/shared/bids-export/",
-        "/gpfs/data/bnc/shared/scripts/fmcdona4/bidsmap.json"
-    ]
-
-    # Run compileArgumentList()
-    x2b_param_list, slurm_param_list, bindings = compile_argument_list(session, test_params, user)
-    assert x2b_param_list == x2b_validation_set
-    assert slurm_param_list == slurm_validation_set
-    assert bindings == bindings_validation_set
+
+        self.assertEqual(x2b_param_list, x2b_validation_set)
+        self.assertEqual(bindings, bindings_validation_set)
+
+        # Run compile_slurm_list()      
+        slurm_param_list = compile_slurm_list(test_params, user)
+
+        self.assertEqual(slurm_param_list, slurm_validation_set)
+
+
+    def test_parse_x2b_params(self):
+        # Define test input data
+        xnat2bids_dict = {
+            "bids_root": "/path/to/bids",
+            "includeseq": ["t1w", "t2w"],
+            "export-only": True,
+            "verbose": 3,
+        }
+        session = "test_session"
+        bindings = []
+
+        # Call the function being tested
+        result = parse_x2b_params(xnat2bids_dict, session, bindings)
+
+        # Define the expected output
+        expected_result = [
+            "test_session",
+            "/path/to/bids",
+            "--includeseq t1w --includeseq t2w",
+            "--export-only",
+            "--verbose",
+            "--verbose",
+            "--verbose",
+        ]
+        expected_bindings = ["/path/to/bids"]
+
+        # Check that the result matches the expected output
+        self.assertEqual(result, expected_result)
+        self.assertEqual(bindings, expected_bindings)
+
+    def test_extract_params_list(self):
```

Comment:
Would it be worth testing both the long and short versions of the params, e.g `--includeseq` and `-i` . Maybe mixing them is enough?